pkg/provider/gcp.go

```diff
@@ -40,6 +48,11 @@ var gcpSurvey = []*survey.Question{
 		Prompt:   &survey.Input{Message: "Enter the name of its gcp project"},
 		Validate: utils.ValidateAlphaNumeric,
 	},
+	{
+		Name:     "region",
+		Prompt:   &survey.Input{Message: "What region will you deploy to?", Default: "us-east1"},
```

Comment:
We should make this a Select input, and you can probably just repurpose the list here for supported regions: https://github.com/pluralsh/plural/blob/master/www/src/components/shell/cloud/gcp.js#L9

Reply:
(we'll want to do this for the other clouds too tbh)